### PR TITLE
Fix array size + New way to check flags.

### PR DESCRIPTION
**Pull request by Hexer10**

### CODE REVIEW

File: addons/sourcemod/scripting/MyJailbreak/playertags.sp [hunk 1]
```diff
@@ -268,40 +268,50 @@ void LoadPlayerTags(int client)
...
+	if (admin != INVALID_ADMIN_ID)
 	{
-		LogMessage("Flags vor buffer: %s", flags[i]);
-		char buffer[2];
-		Format(buffer, sizeof(buffer), flags[i]);
-		LogMessage("Flags nach buffer: %s", buffer);
-
-		if (kvMenu.JumpToKey(buffer, false))
+		char sGroup[32];
+		admin.GetGroup(0, sGroup, sizeof(sGroup));
```
Comment (Hexer10) [inline addons/sourcemod/scripting/MyJailbreak/playertags.sp]: Not sure if 0 it's ok or if it's needed to loop [GroupCount](https://sm.alliedmods.net/new-api/admin/AdminId/GroupCount) times.

Comment (shanapu) [reply addons/sourcemod/scripting/MyJailbreak/playertags.sp]: When I understand description on API docs right, this int is a wildcard.
if the 0 should not work maybe use it with a variable. I don't think we need the GroupCount here.
```
int iBuffer;
admin.GetGroup(iBuffer, sGroup, sizeof(sGroup));
```

Comment (Hexer10) [reply addons/sourcemod/scripting/MyJailbreak/playertags.sp]: I've already used this function on other plugins, and there I used "0", from what I understood "0" is the index of the group, like if the client is in the group *Mod* and in the group *Admin*, the index of one group will be 0 and of the second group would be 1.
I mean something like:
```
	...
	for (int i = 0; i <= admin.GroupCount)
	{
		char sGroup[32];
		admin.GetGroup(i, sGroup, sizeof(sGroup));
		...
	}
```

Comment (shanapu) [reply addons/sourcemod/scripting/MyJailbreak/playertags.sp]: uhhmm... true. I guess I must still be on medication.

Yeah, a loop is needed the player can be in more than one group. would be interesting to know how the group index is determined. the order in cfg or alphabetical.

I never worked with groups so far.

**Comment by Hexer10** [comment]
Need some reworks...

**Pull request merged**